src/rules/migration/no-this-org.ts

```diff
@@ -35,10 +35,15 @@ export const noThisOrg = ESLintUtils.RuleCreator.withoutDocs({
             if (MemberExpressionIsThisDotFoo(node, 'org') && ancestorsContainsSfCommand(context.getAncestors())) {
               // it's ok if there's a this.org on the class...
               const classAbove = getSfCommand(context.getAncestors());
+              if (!classAbove) {
+                return;
+              }
               const runMethod = getRunMethod(classAbove);
+              if (!runMethod) {
+                return;
+              }
```

Comment:
This `runMethod` check could be added to src/rules/migration/no-this-flags.ts also